gpjax/variational_families.py

```diff
@@ -683,7 +683,7 @@ def predict(
         Lz_inv_Kzx = cola.solve(Lz, Kzx)
 
         # A = Lz⁻¹ Kzt / o
```

Comment:
Think pre-commit stuff has gone rogue 😱. The comment should has a `σ` not an `o`.